webapp/src/selectors.js

```diff
@@ -24,9 +24,10 @@ export const getMessage = (state) => {
     }
     return post.message;
 };
-export const getIssues = (state) => getPluginState(state).issues;
-export const getInIssues = (state) => getPluginState(state).inIssues;
-export const getOutIssues = (state) => getPluginState(state).outIssues;
+export const getIssues = (state) => getAllIssues(state).my;
```

Comment:
Can we rename this to `getMyIssues`?

```suggestion
export const getMyIssues = (state) => getAllIssues(state).my;
```